static/app/views/replays/detail/console/consoleMessage.tsx

```diff
@@ -75,13 +83,42 @@ function ConsoleMessage({breadcrumb, isLast}: ConsoleMessageProps) {
       <Icon isLast={isLast} level={breadcrumb.level}>
         {ICONS[breadcrumb.level]}
       </Icon>
-      <Message isLast={isLast} level={breadcrumb.level}>
-        <MessageFormatter breadcrumb={breadcrumb} />{' '}
-      </Message>
+      <MessageWrapper>
+        <Message isLast={isLast} level={breadcrumb.level}>
+          <MessageFormatter breadcrumb={breadcrumb} />
+        </Message>
+        <StyledConsoleTimestamp isLast={isLast} level={breadcrumb.level}>
+          <Hovercard body={`${t('Relative Time')}: ${relativeTimestamp}`}>
+            {getFormattedDate(breadcrumb.timestamp, 'MMM D, YYYY hh:mm:ss A z', {
+              local: false,
```

Comment:
The UTC/timezone setting should come from `ConfigStore` (specifically `user` property). Can we use `<DateTime>` component here? it handles that for us.